src/components/header/ProfileViewerHeader.js

```diff
@@ -30,16 +30,16 @@ import type { State } from '../../types/reducers';
 import type { Milliseconds, StartEndRange } from '../../types/units';
 
 type Props = {|
-  profile: Profile,
-  className: string,
-  hiddenThreads: ThreadIndex[],
-  threadOrder: ThreadIndex[],
-  selection: ProfileSelection,
-  timeRange: StartEndRange,
-  zeroAt: Milliseconds,
-  changeThreadOrder: typeof changeThreadOrder,
-  addRangeFilterAndUnsetSelection: typeof addRangeFilterAndUnsetSelection,
-  updateProfileSelection: typeof updateProfileSelection,
+  +profile: Profile,
+  +className: string,
+  +hiddenThreads: ThreadIndex[],
+  +threadOrder: ThreadIndex[],
+  +selection: ProfileSelection,
+  +timeRange: StartEndRange,
+  +zeroAt: Milliseconds,
+  +changeThreadOrder: typeof changeThreadOrder,
+  +addRangeFilterAndUnsetSelection: typeof addRangeFilterAndUnsetSelection,
+  +updateProfileSelection: typeof updateProfileSelection,
```

Comment:
It looks like an unrelated change  given there is no other change ?

Reply:
It's related in so far as I was in the file for this patch, and updating it to our latest practices.